helix-front/pom.xml

```diff
@@ -154,6 +154,22 @@
         </executions>
       </plugin>
 
+      <plugin>
+	      <groupId>org.apache.maven.plugins</groupId>
+	      <artifactId>maven-surefire-plugin</artifactId>
+	      <executions>
+		      <execution>
+            <id>execution-name</id>
+            <configuration combine.self="override">
+              <suiteXmlFiles>
+                <suiteXmlFile>helix-common/src/test/conf/testng.xml</suiteXmlFile>
```

Comment:
First, why do we reference the config file in helix-common?
Second, can we just configure to skip the maven test in helix-front, since there is no maven test anyway?

Reply:
You may try the following in helix-front/pom.xml and test with `mvn test` in module dir.

```
<plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
           <configuration>
               <skipTests>true</skipTests>
           </configuration>
       </plugin>
```